src/Sylius/Bundle/OrderBundle/NumberGenerator/SequentialOrderNumberGeneratorInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is a part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\OrderBundle\NumberGenerator;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+interface SequentialOrderNumberGeneratorInterface
```

Comment:
This interface should be ~~Sequential~~OrderNumberGeneratorInterface, `Sequential` is specific implementation.